motr/examples/example1.c

```diff
@@ -36,7 +36,8 @@
  * Please change the configuration according to you development environment.
  *
  * How to run:
- * LD_LIBRARY_PATH=/work/cortx-motr/motr/.libs/                              \
+ * LD_LIBRARY_PATH=/work/cortx-motr/motr/.libs/                              
+ * export LD_LIBRARY_PATH
  * ./example1 172.16.154.179@tcp:12345:34:1 172.16.154.179@tcp:12345:33:1000 \
  *         "<0x7000000000000001:0>" "<0x7200000000000001:64>" 12345670
```

Comment:
Original, my intention was to put these in the same line. 
Please refer to the back slash `\` at the end of LINE 39.

It is better to export it, so no need to add " LD_LIBRARY_PATH=/work/cortx-motr/motr/.libs/" at every run.

Reply:
I see, so you had `export LD_LIBRARY_PATH` in your system by default. However, when we install CORTX on the clean server (Emulab or Chameleon testbed cluster), we need to install all the dependency manually. Perhaps, adding a note saying that the `LD_LIBRARY_PATH` must be exported would be great. What do you think?

Reply:
Developers are encouraged to do this by his own if he'd like to export globally LD_LIBRARY_PATH.
Explanation is already in the code itself.